src/main/java/com/example/bookstore/model/OrderItem.java

```diff
@@ -0,0 +1,43 @@
+package com.example.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Entity
+@Data
+@RequiredArgsConstructor
+@Table(name = "order_items")
+@SQLDelete(sql = "UPDATE order_items SET is_deleted = true WHERE id = ?")
+@Where(clause = "is_deleted = false")
+@ToString(exclude = "Order")
+@EqualsAndHashCode(of = "id")
```

Comment:
Too many repeating annotations. Check what is @Data. 
```suggestion
@Data
@RequiredArgsConstructor
@Table(name = "order_items")
@SQLDelete(sql = "UPDATE order_items SET is_deleted = true WHERE id = ?")
@Where(clause = "is_deleted = false")
@ToString(exclude = "Order")
@EqualsAndHashCode(of = "id")
```

Reply:
що мається на увазі? там всі потрбіні анотації